src/dlgConnectionProfiles.cpp

```diff
@@ -1782,14 +1816,18 @@ void dlgConnectionProfiles::loadProfile(bool alsoConnect)
             mudlet::self()->packagesToInstallList.append(QStringLiteral(":/mudlet-mapper.xml"));
         } else if (pHost->getUrl().contains(QStringLiteral("3scapes.org"), Qt::CaseInsensitive) || pHost->getUrl().contains(QStringLiteral("3k.org"), Qt::CaseInsensitive)) {
             mudlet::self()->packagesToInstallList.append(QStringLiteral(":/3k-mapper.xml"));
-        } else {
+        } else if ( not pHost->getUrl().contains(QStringLiteral("localhost"), Qt::CaseInsensitive) ) {
             mudlet::self()->packagesToInstallList.append(QStringLiteral(":/mudlet-lua/lua/generic-mapper/generic_mapper.xml"));
         }
+        if (pHost->getUrl().contains(QStringLiteral("localhost"), Qt::CaseInsensitive)) {
+            mudlet::self()->packagesToInstallList.append(QStringLiteral(":/run-tests.xml"));
+        } else {
+            mudlet::self()->packagesToInstallList.append(QStringLiteral(":/send-text-to-all-games.xml"));
```

Comment:
I'd say these packages would be useful to have on the test profile as well

Reply:
I disagree. A test environment should be as clean as possible, to make certain your tests are not being compromised.
If you want to test `send-text-to-all-games` or `default-mapper`, you should either pull those packages into the test profile, or pull the `run-tests` package into a profile that you want to test.
I made an exception with `run-lua`, because it's so useful when debugging test failure.

Reply:
Fair enough.